data.json

```
@@ -766,5 +766,11 @@
     "githubUsername": "Rishabhrathore055",
     "imageUrl": "https://user-images.githubusercontent.com/77366500/115426452-234e8880-a21e-11eb-9e2d-c508acdb921f.jpg",
     "issueNumber": 445
+  },
+    {
```

```suggestion
  {
```
line up with the other brace